network and address_range tests

chkp-orso opened the pull request:
##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Add and edit unit tests for check_point ansible modules

This commit adds several missing unit tests for the check_point ansible
modules.

test_cp_mgmt_network.py
test_cp_mgmt_address_range.py
test_cp_mgmt_address_range_facts.py
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Bugfix Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->
check_point
##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

chkp-orso committed:
Update test_cp_mgmt_network.py

chkp-orso committed:
Merge branch 'temp-b' of https://github.com/chkp-orso/ansible into network-and-address-range-tests

chkp-orso committed:
commit network and address_range

chkp-orso committed:
commit network and address_range

chkp-orso committed:
remove test_cp_mgmt_network from ignore.txt

chkp-orso committed:
remove test_cp_mgmt_network from ignore.txt

s-hertel committed:
remove undefined YAML anchor (#62273)

Glandos committed:
Update mitogen link (#62247)

ganeshrn committed:
Fix for junos cli_config replace option (#62131)

* Fix for junos cli_config replace option

*  For device that support replace option by loading
   configuration from a file on device `config` option
   is not required and value of `replace` option is the
   path of configuration file on device. This fix allows
   invoking run() function in cli_config if `config` option
   is None and `replace` option is not boolean

*  The command to replace running config on junos device
   is `load override <filename>` and not `load replace <filename>`
   This is fixed in the junos cliconf plugin.

* Add integration test

danielmellado reviewed (approved):
Thanks! Just make sure to rebase the remaining patches in order to avoid conflicts!

Qalthos committed:
Standardize eos resource modules (#61736)

* Fix eos_l3_interfaces case sensitivity

* Unify EOS module notes

* Add normalize_interfaces to eos_l2_interfaces

* Pull normalize_interface into eos_interfaces

* Add normalize_interface to lag_interfaces

* Add normalize_interface to lldp_interfaces

* Add normalize_interface to lacp_interfaces

* more module cleanup

* Add changelog

felixfontein committed:
Re-enable some docker_swarm tests. (#61875)

samdoran committed:
Update default test container with Python 3.8b4 (#62100)

sivel committed:
Don't treat no checksum as a checksum match (#62146)

Fixes #61978
* moar tests for get_url fetch behavior with existing file
* add changelog fragment

sivel committed:
Ensure we don't erase unsafe context in TaskExecutor.run on bytes (#62287)

* Ensure we don't erase unsafe context in TaskExecutor.run on bytes. Fixes #62237

* Remove unused import

* Add missing import

* use args splatting for to_unsafe_text/bytes

* Add security issue to changelog

* fix yaml linting issue

n3pjk committed:
Add httpapi VMware REST VmwareRestModule (#60914)

* Initial commit of VMware HttpApi REST

New directory structure created for httpapi-based modules. These will
live in the 'vmware_httpapi' directory under 'modules/cloud'.  The
AnsibleModule class was extended to create the VmwareRestModule class
that lives in the 'vmware_httpapi' directory under 'module_utils'. It
implements comms with the httpapi plugin, and also provides debugging
output, url and filtering generation by object, and support for
multiple VMware REST APIs. It also provides dynamic handling of HTTP
return codes that can be tailored to each module's needs.

flowerysong committed:
Add version 2.14 to the removed_in list (#62298)

devel is now 2.10, so this needs to be bumped as well.

samccann committed:
revert back to contiguous (#62301)

goneri committed:
azure: re-balance tests to avoid timeout

- azure_rm_azurefirewall currently takes more than 25m to run: #62307
- azure_rm_manageddisk takes around 2 minutes

mattclay committed:
Fix display of sanity test doc links.

The documentation links are now displayed when running from an install.

Previously the links were only displayed when running from source.
This was due to ansible-test checking for the presence of documentation files locally, which are only present when running from source.
The check is no longer necessary since there is a sanity test in place to enforce the presence of documentation for all sanity tests.

chkp-orso committed:
Add unit tests for check_point ansible modules (#62214)

* Update test_cp_mgmt_network.py

* 18 tests

chkp-orso committed:
Third pr 18 tests (#62215)

* Update test_cp_mgmt_network.py

* 18 tests

ctrufan committed:
Add ecs_domain module (#62007)

* Add ecs_domain module

* Fixes to integration tests and module

* Fixes to tests and module

* Corrections to revalidation behavior, cna only revalidate domains in expiring.

* Remove debugs for final test run, fix sanity check test fails.

* Add checks for domain status

* Add changelog fragment for new module.

* Removed extra space in backtick

* Minor fixes to make behavior more consistent and correct documentation.

* Update lib/ansible/modules/crypto/entrust/ecs_domain.py

Co-Authored-By: Felix Fontein <felix@fontein.de>

* Update lib/ansible/modules/crypto/entrust/ecs_domain.py

Co-Authored-By: Felix Fontein <felix@fontein.de>

* Update lib/ansible/modules/crypto/entrust/ecs_domain.py

Co-Authored-By: Felix Fontein <felix@fontein.de>

* Apply suggestions from code review

Co-Authored-By: Felix Fontein <felix@fontein.de>

* Change casing of verification method enum, remove redundant changelog fragment

* Return ov_eligible and ev_eligible fields even if false, as long as they're returned by ECS API

felixfontein committed:
Remove deprecated letsencrypt alias of acme_certificate (#61648)

briantist committed:
Fix check mode support in win_group (#61977)

* Fix check mode support in win_group

* Fix copy/paste error

joshinryz committed:
Support large folder size comparisons for win_find (#58466)

* Support large folder size comparisons for win_find

Changed [int] to [int64] to support larger folders. Otherwise module fails as soon as a large folder is encountered.

* Create 58466-FIX_win_find-Bug-Get-FileStat_fails_on_large_files.yml

ShachafGoldstein committed:
win_find - Fix deduped files mistaken for directories (#58680)

* win_find - Fix deduped files mistaken for directories

* Update win_find.ps1

* Update win_find.py

* Update win_find.ps1

jn-bedag committed:
correct behaviour of win_iis_webapplication when no applicationpool is specified (#61227)

* correct behaviour of win_iis_webapplication when no applicationpool is specified

* documentation for win_iis_webapplication apppool change

ShachafGoldstein committed:
win_format - Idem not working if file exist but same fs (#59819)

* win_format - Idem not working if file exist but same fs

* Test fix

* Fix test assertion syntax

* Update tests.yml

Andersson007 committed:
Fix typos in remote_management modules (#62336)

ganeshrn committed:
Change enable to enabled for junos_interfaces module (#62321)

Fixes #62319

Change `enable` option to `enabled` in junos_interfaces
and junos_lldp_interfaces
data model to be in sync with other network platform
resource modules added in 2.9 version.

chkp-orso committed:
Merge branch 'temp-b' of https://github.com/chkp-orso/ansible into network-and-address-range-tests

Akasurde commented:
Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/CheckPointSW/CheckPointAnsibleMgmtCollection

If you have further questions please stop by IRC or the mailing list:

* IRC: #ansible on irc.freenode.net
* mailing list: https://groups.google.com/forum/#!forum/ansible-project

Thanks.
